constants/constants.go

```diff
@@ -31,6 +31,9 @@ const (
 	// CertificatePEMBlock is the name of the PEM block where certificate is stored
 	CertificatePEMBlock = "CERTIFICATE"
 
+	// AnonymizationKeyPEMBlock is the name of the PEM block where anonymization key is stored
+	AnonymizationKeyPEMBlock = "ANONYMIZATION KEY"
```

Comment:
Should we be even more specific?
```suggestion
	AnonymizationKeyPEMBlock = "TELEPORT ANONYMIZATION KEY"
```